packages/kotti-ui/source/kotti-comment/components/CommentActionsOptions.vue

```diff
@@ -24,13 +22,5 @@ export default defineComponent<{
 	props: {
 		options: { type: Array, required: true },
 	},
-	components: {
-		KtPopover,
```

Comment:
Why?

Reply:
accidental, fixed on the MR where i brought back all the internal imports but forgot to rebase properly i guess